fs.go

```diff
@@ -45,17 +47,18 @@ func IsDir(name string) (bool, error) {
 	return true, nil
 }
 
-func IsNonEmptyDir(name string) (bool, error) {
-	isDir, err := IsDir(name)
```

Comment:
Seems like we still need this case on Windows; I guess it's not an error to `ioutil.ReadDir` on a file there.